package.json

```diff
@@ -91,10 +91,13 @@
     "liquid-fire": "0.27.3",
     "loader.js": "4.0.1",
     "pagination-pager": "2.4.2",
-    "postcss": "5.1.0"
+    "postcss": "5.1.0",
+    "unicode-byte-truncate": "^1.0.0"
   },
   "bugs": {
     "url": "https://github.com/CenterForOpenScience/ember-osf-preprints/issues"
   },
-  "homepage": "https://github.com/CenterForOpenScience/ember-osf-preprints#readme"
+  "homepage": "https://github.com/CenterForOpenScience/ember-osf-preprints#readme",
+  "dependencies": {
```

Comment:
If there isn't anything in `dependencies`, do we still need to include it in here?